.env.example

```diff
@@ -0,0 +1,13 @@
+AM_URL=<<<URL to your AM instance>>>
```

Comment:
Not sure what the comment syntax is for this file type, but should we comment what the `env` file affects? As it is, it looks like it might control _All The Things_ (™) in the repo – which I know may be a goal, but it doesn't yet.

Reply:
Yes, perhaps we could comment in here - or mention in the project README, that .env specifies the environment for samples? I think the syntax is with "#", so `# this is a comment`

Reply:
What do you mean 'it doesn't yet'?

The syntax here is just placeholder (see removal diff), it was the same.

We can update it to a different placeholder syntax if preferred or none.